content/en/blog/2023-11-modelix-platform-release-23.2/index.md

```diff
@@ -0,0 +1,97 @@
+- [Development Insights](#development-insights)
+  - [New Approach to MPS-Related Components](#new-approach-to-mps-related-components)
+  - [Contribution Guide](#contribution-guide)
+  - [Marking Experimental Features](#marking-experimental-features)
+  - [Extending Automation](#extending-automation)
+- [What's To Come](#whats-to-come)
+
+# Feature Highlights
+
+## ModelQL
+
+The [ModelQL query language](https://docs.modelix.org/modelix/23.2/core/explanation/modelql.html) provides a dynamic way of loading parts of the model on demand while reducing the number of requests to a minimum.
+
+## Model Client as NPM Package (experimental)
+
+The previously existing [model client](https://github.com/modelix/modelix.core/tree/23.2/model-client) is now also published as [`@modelix/model-client` NPM package](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/).
+
+The exposed API is minimal and experimental. 
+In the future, we want to make more features from the Kotlin model client directly accessible from the exposed API in the NPM package.
+
+## Vue.js Bindings (experimental)
+
+Built upon [`@modelix/model-client`](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/) we created a library to interact with models through [Vue.js](https://vuejs.org/).
+
+With [`@modelix/vue-model-api`](https://docs.modelix.org/modelix/23.2/core/reference/component-vue-model-api.html) we are currently building sample projects with Vue.js.
+After that, we want to stabilize the API and show you how you can use it for your projects.
+
+## MPS Model Adapters
+
+We created [adapters](https://github.com/modelix/modelix.core/tree/23.2/mps-model-adapters) that serve as an interface between MPS and the modelix [model-api](https://docs.modelix.org/modelix/23.2/core/reference/component-model-api.html).
+They are compatible with all MPS versions that modelix currently supports.
+
+
+## Bulk Model Synchronization Plugin (experimental)
+
+The [bulk-model-sync Gradle Plugin](https://docs.modelix.org/modelix/23.2/core/reference/component-bulk-model-sync-gradle.html) allows bulk synchronization of models between MPS and a [model-server](https://docs.modelix.org/modelix/23.2/core/reference/component-model-server.html).
+The plugin applies changes incrementally, and can be run in existing CI systems to schedule and run the synchronization in both directions.
+This way you can implement a custom Git Integration tailored to your needs.
+
+The plugin is considered unstable, but will be stabilized soon.
+
+# Development Insights
+
+## New Approach to MPS-Related Components
+
+
+Up until now, we built and released MPS-specific components for each MPS version individually.
+Going forward, we want to release single components, which are compatible with all MPS versions, that are currently supported by modelix.
```

Comment:
```suggestion
Going forward, we want to release single components, which are compatible with all MPS versions that are currently supported by modelix.
```

https://www.merriam-webster.com/grammar/when-to-use-that-and-which I have to read that up from time to time again because the rules are so unintuitive for Germans

Reply:
Done